src/runtime/virtcontainers/pkg/oci/utils.go

```diff
@@ -442,6 +443,20 @@ func addHypervisorConfigOverrides(ocispec specs.Spec, config *vc.SandboxConfig)
 			config.HypervisorConfig.EntropySource = value
 		}
 	}
+	if epcSize, ok := ocispec.Annotations[vcAnnotations.SGXEPC]; ok {
+		quantity, err := resource.ParseQuantity(epcSize)
+		if err != nil {
+			return fmt.Errorf("Couldn't parse EPC '%v': %v", err, epcSize)
+		}
+
+		if quantity.Format != resource.BinarySI {
+			return fmt.Errorf("Unsupported EPC format '%v': use Ki | Mi | Gi | Ti | Pi | Ei as suffix", epcSize)
+		}
```

Comment:
If parsing is OK, this this error ever going to trigger?

Reply:
Here I'm trying to force the BinarySI format, otherwise DecimalSI format can be used and that's not expected

Reply:
OK, I did not check how cloud-hypervisor handles this. This annotation isn't usually set manually but by the webhook which gives: `quantity := resource.NewQuantity(totalEpc, resource.BinarySI)` . This validation probably makes sense anyway.  However, note that 1024 should be as good as `1Ki` and results in the same value after `AsInt64()`.